stdlib/Random/src/XoshiroSimd.jl

```diff
@@ -0,0 +1,350 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+module XoshiroSimd
+# Getting the xoroshiro RNG to reliably vectorize is somewhat of a hassle without Simd.jl.
+import ..Random: TaskLocal, rand, rand!, UnsafeView, Xoshiro, SamplerType
+
+
+
+#Vector-width. Influences random stream. We may want to tune this before merging.
+xoshiroWidth() = Val(8)
```

Comment:
The capitalization in this file seems a bit off (camelCase). Is that intentional?